modules/ROOT/pages/cloudhub-networking-guide.adoc

```diff
@@ -40,22 +34,21 @@ Here is an example of a Mule configuration that utilizes this to expose an HTTPS
 </http:listener-config>
 ----
 
-You may also refer to this knowledge-base article: https://support.mulesoft.com/s/article/Sample-App-Configuring-HTTPS-endpoint-for-deployment-in-Cloudhub[Sample App: Configuring HTTPS endpoint for deployment in CloudHub]
-
-*Important*: On the Mule worker, the CloudHub load balancer proxies port :80 to :8081 for HTTP and proxies port :443 to :8082 for HTTPS. +
-The `http.port` property is automatically set to port 8081 for HTTP, and `https.port` is set to port 8082 for HTTPS. If other values for http.port and https.port are specified in the mule-app.properties file, these are overwritten at deployment time.
+[IMPORTANT]
+On the Mule worker, the CloudHub load balancer proxies port :80 to :8081 for HTTP and proxies port :443 to :8082 for HTTPS. +
+The `http.port` property is automatically set to port 8081 for HTTP, and `https.port` is set to port 8082 for HTTPS. If other values for `http.port` and `https.port` are specified in the `mule-app.properties` file, these are overwritten at deployment time.
 
 [NOTE]
 --
-For each request that a client makes through CloudHub's load balancer (_myapp.cloudhub.io_), the load balancer maintains two connections. One connection ​from​ the client and one to your Worker.  For each connection, the load balancer manages an idle timeout of 300 seconds that is triggered when no data is sent over ​either​ connection.  If no data has been sent or received during this time period, the load balancer closes ​both connections.
+For each request that a client makes through CloudHub's load balancer (_myapp.cloudhub.io_), the load balancer maintains two connections. One connection ​is from​ the client and one is to your worker.  For each connection, the load balancer manages an idle timeout of 300 seconds that is triggered when no data is sent over ​either​ connection.  If no data is sent or received during this time period, the load balancer closes ​both connections.
```

Comment:
should _myapp.cloudhub.io_ always have tick marks (`_myapp.cloudhub.io_`) or do we leave them off because the underscore character makes trouble with tick marks?